xknx/prog/device.py

```diff
@@ -0,0 +1,344 @@
+"""This module implements the application layer functionality of a device."""
```

Comment:
Remove all print() calls in this file. If the prints are needed use a logger and log in an appropriate log level